src/migration_utilities.js

```diff
@@ -9,4 +9,14 @@ const argv = require("yargs")
   .version(false)
   .argv
 
-module.exports = argv
\ No newline at end of file
+function isMigrationRequired(network) {
+  if (network === "development" || network == "coverage" || network == "developmentdocker" || network == "development-fork") {
```

Comment:
What about just returning this check instead of the `if/else`?
Also, would you mind doing the strict comparison (`===`) for all values?